Frameworks/Foundation/NSURLSessionConfiguration.mm

```diff
@@ -41,22 +41,13 @@ - (instancetype)_initWithSharedDefaults {
     return self;
 }
 
-- (instancetype)_initTLSAndCache {
-    if (self = [self _initWithSharedDefaults]) {
-        _TLSMaximumSupportedProtocol = kTLSProtocol12;
-        _TLSMinimumSupportedProtocol = kTLSProtocol1;
-        _URLCache = [[NSURLCache sharedURLCache] retain];
-    }
-    return self;
-}
-
 /**
  @Status Interoperable
 */
 - (instancetype)init {
     if (self = [self _initWithSharedDefaults]) {
         _TLSMaximumSupportedProtocol = kTLSProtocol12;
-        _TLSMinimumSupportedProtocol = kSSLProtocol3;
+        _TLSMinimumSupportedProtocol = kTLSProtocol1;
```

Comment:
It is also kTLSProtocol1 on iOS

---
In reply to: [73745172](https://github.com/Microsoft/WinObjC/pull/828#discussion_r73745172) [](ancestors = 73745172)